packages/entity/src/EntityQueryContextProvider.ts

```diff
@@ -23,24 +24,26 @@ export default abstract class EntityQueryContextProvider {
   /**
    * Vend a transaction runner for use in runInTransactionAsync.
    */
-  protected abstract createTransactionRunner<T>(): (
-    transactionScope: (queryInterface: any) => Promise<T>
-  ) => Promise<T>;
+  protected abstract createTransactionRunner<T>(
+    transactionConfig?: TransactionConfig
+  ): (transactionScope: (queryInterface: any) => Promise<T>) => Promise<T>;
 
   protected abstract createNestedTransactionRunner<T>(
-    outerQueryInterface: any
+    outerQueryInterface: any,
+    transactionConfig?: TransactionConfig
```

Comment:
Is this parameter needed? Do we want to support this since Postgres doesn't support nested transactions (just savepoints)?